static/app/components/acl/demoModeDisabled.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {isDemoModeEnabled} from 'sentry/utils/demoMode';
+
+import DisableInDemoMode from './demoModeDisabled'; // Adjust the import path as necessary
+
+jest.mock('sentry/utils/demoMode', () => ({
+  isDemoModeEnabled: jest.fn(),
+}));
+
+jest.mock('sentry/locale', () => ({
+  t: jest.fn(key => key), // Mock translation function
+}));
+
+describe('DisableInDemoMode', () => {
+  it('renders children when demo mode is disabled', () => {
+    (isDemoModeEnabled as jest.Mock).mockReturnValue(false);
```

Comment:
typescript friendly option
```suggestion
    jest.mocked(isDemoModeEnabled).mockReturnValue(false);
```